packages/webapp/nginx.conf

```diff
@@ -23,6 +23,7 @@ http {
     log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                       '$status $body_bytes_sent "$http_referer" '
                       '"$http_user_agent" "$http_x_forwarded_for"';
+    server_tokens off;
```

Comment:
@iperdomo My question here is: 

Is http context enough? Or should this also be present in the server context?

Reply:
There is a hierarchy `http` > `server` > `location` _context_, if something is defined at `http` applies to all child contexts unless there is more specific configuration.

* https://www.digitalocean.com/community/tutorials/understanding-the-nginx-configuration-file-structure-and-configuration-contexts
* https://nginx.org/en/docs/http/ngx_http_core_module.html#http
* https://nginx.org/en/docs/http/ngx_http_core_module.html#server
* https://nginx.org/en/docs/http/ngx_http_core_module.html#location